test/react/login_form.test.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-undef */
+import * as React from "react";
+import LoginForm from "../../app/javascript/components/login_form";
+import { expect, test } from "@jest/globals";
+import { fireEvent, getByRole, queryByTestId, render, RenderResult } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import "@testing-library/jest-dom";
+import mockAxios from "jest-mock-axios";
+
+const renderForm = (
+  path = "/log",
+  token = "token"
+) : RenderResult => {
+  return render(
+        <LoginForm path = {path} token = {token}/>
+  );
+};
+
+describe("Describe the login form", () => {
```

Comment:
Lets take some time to talk through the react tests here!

Reply:
I've added the tests that we discussed